cmd/root.go

```diff
@@ -26,64 +26,75 @@ type Scanner interface {
 // log level
 var level logconfig.Level
 
-// print result of scan as JSON to stdout
-var printJSON bool
+// format of logs to stdout & stderr
+var logFormat string
+
+// list of supported format of logs to stdout & stderr
+var logFormatSupportedFormats = [1]string{"json"}
```

Comment:
Shouldn't we have also a `text` format, which would be the current one?

Reply:
but, the current one outputs a json per line (see the output on previous comment). Maybe I'm misunderstanding here.

Reply:
Prior to this PR, the output was not JSON-encoded, am I right? It was something like 
```
DATETIME - WARN - foo bar
```

This would be the default format to use, the `text` one.

To go back to policy-server, when no `--log-fmt` is not provided the log is in "human"/text mode. A different format can be picked by specifying `--log-fmt <FMT>`. This is how audit-scanner will work once this PR is merged. I just propose to add `text` as one of the possible values of `log-fmt`

Reply:
no, the output was always JSON, for the logs from zerolog, or the scan output.

zerolog seems to only support JSON or a binary format. This would apply to normal logs.

For the scan result, I'm afraid that it can be quite verbose for logs, so IMHO it should be disabled by default. That's the current behaviour and the one from this PR.
Before this PR one needed to do `--print` to print the result of the scan in JSON.
With this PR, one needs to do `--log-fmt=json` to print the result of the scan in JSON. We could add other supported formats to the scan output, but to have the same output format for normal logs, we would need to refactor the use of zerolog of find an alternative.

Reply:
Sorry, I really misunderstood the whole thing. As discussed, let's go back to the original implementation (so `--print`), but with a better name of the flag (like `--print-report`). It would be a boolean flag, disabled by default. Once enabled it would print to the stdout the report using as a JSON object

Reply:
Uook, no problem! Renamed from `--print` to `--output-scan` which was the first approach and matches https://github.com/kubewarden/helm-charts/pull/282.